nltk/data.py

```diff
@@ -1111,13 +1140,14 @@ def readline(self, size=None):
             if len(lines) > 1:
                 line = lines[0]
                 self.linebuffer = lines[1:]
-                self._rewind_numchars = len(new_chars)-(len(chars)-len(line))
+                self._rewind_numchars = len(
+                    new_chars) - (len(chars) - len(line))
```

Comment:
Such change makes readability worse, not better: the expression is split at a random point. Have you used some automated tools? 

Instead of splitting long expressions into multi lines it is usually better to introduce local varaibles. Here a variable for `len(chars)-len(line)` can be introduced - what should be its name?